RSEConUK2019-Cylc-Talk/index.html

```diff
@@ -403,165 +404,378 @@
+              style="width: 125px;float:right"
+              src="media/icons/play.svg"
+              alt="play-icon"
+            />
+          <em>simulated<br> time</em><br> cycling
+           </p>
         </div>
-        <div class="threecolumn">
-          <br><br>
-          <p><em>real-time cycling</em></p>
-          <img src="media/icons/clock.svg" alt="clock-face"
-          style="height:200px;position:absolute;top:300px;right:10px">
-          <embed
-            class="flexgrow"
-            src="media/graph/cycling-workflow.svg"
-            height="400"
-          />
+     </div>
+
         </div>
+        </div>
+
+        <div class="flex-container-vert" style="flex-basis: 250px">
+          <p><b>operations</b>
+            <!--
+              'weather symbols' clipart (free to use)
+              http://www.clker.com/clipart-weather-symbols-1.html
+            -->
+            <img
+              style="width:75%"
+              src="media/decorative/weather-symbols.svg"
+              alt="weather-symbols"
+            /></p>
+        </div>
+
       </div>
     </div>
 
-    <div class="step">
-      <div class="grid-container">
-        <div class="grid-item"></div>
-        <div class="grid-item"></div>
-        <div class="grid-item"></div>
-        <div class="grid-item">
-          provides the meta-scheduling for workload on
-          <em>460,000 core HPC</em>
+    <div
+      id="met-office-figures"
+      class="step full vflex"
+      >
+      <div class="flex-container">
+
+
+      <div class="flex-container-vert" style="flex-grow: 3">
+
+          <div style="flex-basis: 100px"></div>
+        <div>
+        demonstrated scaling to:
+          <ul>
+          <li><em>&tilde;1000s</em> of <em>recurrences</em></li><br>
+          <li><em>&tilde;10,000s</em> of <em>tasks</em></li><br>
+          <li><em>&tilde;100,000s</em> of <em>dependencies</em></li><br>
+          </ul>
         </div>
-        <div class="grid-item-central">
-          <img src="media/met-office-brand/metoffice-logo-black.png"
-            alt="metoffice-logo-black"
-            style="height:100px;">
-            (largest Cylc-using site)
         </div>
-        <div class="grid-item">demonstrated<br>
-        <em>scaling to 10,000s of tasks</em>
-        (with caveats)</div>
-        <div class="grid-item"></div>
-        <div class="grid-item">
-        used in <em>both</em> research & operations
+
+        <div class="flex-container-vert" style="flex-basis: 100px">
+            <div>
+              <img src="media/met-office-brand/metoffice-logo-black.png"
+                alt="metoffice-logo-black"
+                style="height:100px">
+            </div>
+        </div>
+
+        <div class="flex-container-vert" style="flex-grow: 2">
+          <div><p>used in <em>both</em><br> research &<br> operations</p></div>
+          <div style="flex-basis: 150px"></div>
+          <div>provides all meta-scheduling for the workload on
```

Comment:
May need to be cautious with wording.

```suggestion
          <div>provides meta-scheduling for the workload on
```

Reply:
:+1: Fair enough, caution is wise.